### PR TITLE
v0.16.0 with updated host req and python_min

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge-admin please rerender

**Comment by thewchan** [comment]
Appreciate a review and merge and other packages depends on this version. Thanks! @MatthieuDartiailh @gabrielcnr @marcelotrevisani

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,19 +16,25 @@ build:
...
 
 test:
   imports:
     - bytecode
-    - bytecode.instr
-    - bytecode.flags
-    - bytecode.concrete
-    - bytecode.cfg
+  commands:
+    - pip check
```
Comment (MatthieuDartiailh) [inline recipe/meta.yaml:34]: Could you quickly explain those changes ?

Comment (thewchan) [reply recipe/meta.yaml:34]: Just following the usual patterns with the test section adding `pip check` for dependency checking. We can add back to sub-package import test but in my experience those tend to change breaking the test in the future

Comment (thewchan) [reply recipe/meta.yaml:34]: See: https://conda-forge.org/docs/maintainer/adding_pkgs/#test

Comment (thewchan) [reply recipe/meta.yaml:34]: @MatthieuDartiailh anything else needed here?

**Pull request merged**